.gitignore

```diff
@@ -76,7 +76,7 @@ src/Mocks/Editor/project.json
 src/Debugger/Impl/project.json
 src/R/Support/Impl/project.json
 src/R/Sql/Impl/project.json
-
+src/R/Wpf/Impl/project.json
```

Comment:
Yep. Apparently it was added some time ago, but project.json wasn't removed accordingly, so it kept popping up in the changelist and annoying me :)